### ISSUE TITLE
Forging node setup - Docker env variables parsing fails

### ISSUE THREAD

**Issue by JesusTheHun** [LiskHQ/lisk-core#222]
Following the documentation to setup a node using docker works just fine, until the forging point.

I’ve updated the `docker-compose.override.yml` following the `README.md` and started my containers using `docker-compose -f docker-compose.yml -f docker-compose.override.yml up -d`

When I do :
```
GET http://localhost:7000/api/node/status => OK
GET http://localhost:7000/api/node/status/forging => KO : {“message”:“Access Denied”}
```

From the node itself (inside the container itself using docker exec) or from the outside.
Logs show this error :

> lisk_1 | CheckIpInList: TypeError: cidrString.split is not a function

As you can see [here](https://github.com/indutny/node-ip/blob/master/lib/ip.js#L219), it comes from the `ip.cidrSubnet` function used [in the SDK](https://github.com/LiskHQ/lisk-sdk/blob/v2.3.7/framework/src/modules/http_api/helpers/check_ip_in_list.js#L44).

So basically something is passed to this function, and it’s not a string, so it fails.

`docker-compose.override.yml`
> version: "3"
>services:
>  lisk:
>    environment:
>      - LISK_API_PUBLIC=true
>      - 'LISK_FORGING_DELEGATES=65a2...847fb|iterations=1000000&cipherText=...&version=1'
>      - 'LISK_API_WHITELIST=127.0.0.1,172.17.0.1,172.17.0.3,MY_HOME_IP'
 >     - 'LISK_FORGING_WHITELIST=127.0.0.1,172.17.0.1,172.17.0.3,MY_HOME_IP'

Where `MY_HOME_IP` is my actual home ip, hardcoded.

**Comment by JesusTheHun** [comment]
The issue has been identified and resolved by https://github.com/LiskHQ/lisk-sdk/pull/4816
